design-toolkit/components/src/components/table/index.tsx

```diff
@@ -277,6 +288,7 @@ export function Table<T extends { id: Key }>({
       return row.id ? row.id.toString() : index.toString();
     },
     enableRowSelection: true,
+    manualSorting: manualSorting,
```

Comment:
if continuing with the docs way of pinning, add: 

```suggestion
    enableRowPinning: true,
    manualSorting: manualSorting,
    onRowPinningChange: setRowPinning,
```